src/verifai/scenic_server.py

```diff
@@ -62,3 +73,160 @@ def _simulate(self, scene):
 
     def terminate(self):
         pass
+
+class DummySampler(VerifaiSampler):
+
+    def nextSample(self, feedback):
+        return self.last_sample, None
+
+@ray.remote
+class SampleSimulator():
+
+    def __init__(self, scenic_path, worker_num, monitor, options={}, use_carla=False,
+    scenario_params={}):
+        print(scenario_params)
+        scenario_params.update({
+            'port': 2000 + 2*worker_num
+        })
```

Comment:
Overriding a global parameter named "port" with this particular value needs to be documented.